### PR TITLE
pbench directory

### PR DESCRIPTION
**Pull request by dvalinrh**

### CODE REVIEW

**Comment by dvalinrh** [comment]
Add sub directory for copy

**Comment by frival** [comment]
Add a note in the PR for what this is supposed to solve so we can remember it down the road.  Other than that LGTM.

File: execute_pbench [hunk 1]
```diff
@@ -14,6 +14,7 @@ spacing=0
 sys_type=""
 test_name=""
 times_to_run=1
+results_dir=""
```
Comment (frival) [inline execute_pbench:17]: This should really be pbench_results_dir since it's _only_ taking effect for pbench tests.

**Review by frival — APPROVED** [review]
Fix the naming then LGTM.

**Comment by dvalinrh** [comment]
If you do not provide a --results_dir, we will place the results under the system name.  If --results_dir is designated, then we will create the directory under the system name and place the data there.  This was done so we do not end up with all the io_burst tests directly under the machine name when we are rebooting between each test.

**Comment by dvalinrh** [comment]
Merging code with --results_dir option so we can create another sub directory under the system name.  This allows us to keep the directories and test better organised in case of io_burst test and we are rebooting the system.

**Pull request merged**